bundles/core/src/main/java/com/adobe/cq/forms/core/components/internal/models/v1/formsportal/SearchAndListerImpl.java

```diff
@@ -86,7 +86,7 @@ public String getTitle() {
     @Override
     public String getLayout() {
         if (StringUtils.isEmpty(layout)) {
-            return "Card";
+            return "card";
```

Comment:
create constants

Reply:
Shifted this to a provider type constant interface in PortalLister Interface